runtime/src/main/cpp/Runtime.cpp

```diff
@@ -78,10 +79,14 @@ RuntimeState* initRuntime() {
   RuntimeState* result = konanConstructInstance<RuntimeState>();
   if (!result) return nullptr;
   result->memoryState = InitMemory();
+  bool firstRuntime = atomicAdd(&aliveRuntimesCount, 1) == 1;
   // Keep global variables in state as well.
-  InitOrDeinitGlobalVariables(INIT_GLOBALS);
-  konan::consoleInit();
-  atomicAdd(&aliveRuntimesCount, 1);
+  if (firstRuntime) {
+    konan::consoleInit();
+    InitOrDeinitGlobalVariables(INIT_GLOBALS);
+  } else {
+    InitOrDeinitGlobalVariables(INIT_THREAD_LOCAL_GLOBALS);
```

Comment:
`INIT_THREAD_LOCAL_GLOBALS` appears to be asymmetric with `DEINIT_THREAD_LOCAL_GLOBALS`:
the former is not called for the first runtime, while the latter is called for the last runtime.